website/docs/r/schema_site_service_graph.html.markdown

```diff
@@ -36,6 +42,11 @@ resource "mso_schema_site_service_graph" "example" {
 * `service_graph_name` - (Required) The name of the Service Graph.
 * `service_node` - (Required) List of service nodes attached to the Site Service Graph. Maintaining the order of the service nodes is essential.
     * `device_dn` - (Required) Dn of device associated with the service node of the Service Graph.
+    * `provider_connector_type` - (Optional) Connector type of the provider type of the service node. Allowed values are `redir` and `none`. This parameter is only applicable for third_party_load_balancer and third-party firewall service nodes, when the template is attached to cloud sites.
```

Comment:
when firewall is separate input should this be mentioned in provider_connector_type?